src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -217,6 +217,30 @@ def _lock_cb(cls, mode, n, file, line):
             )
 
 
+def _verify_openssl_version(version):
+    if version < 0x10000000:
+        if os.environ.get("CRYPTOGRAPHY_ALLOW_OPENSSL_098"):
+            warnings.warn(
+                "OpenSSL version 0.9.8 is no longer supported by the OpenSSL "
+                "project, please upgrade. The next version of cryptography "
+                "will completely remove support for it.",
+                utils.DeprecatedIn12
```

Comment:
This should probably become `DeprecationWarning` because the other instance of `DeprecatedIn12` is going to be removed in 1.4 per our standard policy.